### ISSUE TITLE
New ID Reference - publication may refer to

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2554]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Goal**
In the NMMNH:Paleo collection we have a specimen that was published with an incorrect catalog number. (NMMNH:Paleo:3870 was published as NMMNH:Paleo:3810, but both records exist in the datbase) I'd like to make it easy for those who search NMMNH:Paleo:3810 to find the correct (NMMNH:Paleo:3870) published specimen by providing a link directly to it.

**Context**
Current relationships don't describe a link due to a publication error.

**Table**
https://arctos.database.museum/info/ctDocumentation.cfm?table=CTID_REFERENCES

**Value**
publication may refer to

**Definition**
A catalog number which references a cataloged item that was incorrectly identified in a publication with the catalog number of the cataloged item. (OK that is super cruddy, but I cannot think of a way to say this well! Help!)

**Priority**
Please assign a priority-label.

**Comment by dustymc** [comment]
I don't think this is WRONG (although the value seems overly-specific), but it does seem like more structure than we really need for this use case.

I suggest some type ("published identifier" is nice, even though I still don't know where it came from!) with the actionable identifier (http://arctos.database.museum/guid/NMMNH:Paleo:3870) as the value. In conjunction with the 'erroneous citation' that you should create I think that's sufficient to get people where they need to be.

**Comment by Jegelewicz** [comment]
Yeah, but just using the relationship of "self" seems less than optimal and the link also doesn't seem to a very obvious reason even if there is an erroneous citation....

**Comment by dustymc** [comment]
Yes, agreed, it's not "correct" - I just think it's a fringe-enough use case to warrant some discussion. Is a formally correct solution to this worth the extra complexity of a new concept that'll never be used again? Are there many of these just waiting for a solution? What's a "these literally have nothing to do with each other" relationship mean for something like globi? Etc., etc., etc.
